packaging/rpm/rear.spec

```diff
@@ -154,6 +156,9 @@ fi
 %prep
 %setup -q
 
+%check
+%{__make} validate
```

Comment:
Only a question out of curiosity:

Why an indirection via `%{__make}` instead of plain `make`?
Was there ever a case in practice where plain `make` was wrong?

https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/
does not mention RPM's macros for standard commands
so I wonder if RPM macros like `%{__make}` are recommended
(I did not read the whole Fedora Packaging Guidelines).

Reply:
@jsmeix I am not sure. The macro is being used, so @antonvoznia did what the others are doing, see for example https://src.fedoraproject.org/rpms/kernel/pull-request/43. But I have not seen the macro being documented or standardized anywhere.

Reply:
Don't worry - I only asked out of curiosity.

Feel free to merge it as you like.